docs(stack): add related component tips (#7668)
yamada-ui/yamada-ui: www/contents/components/(components)/stack.mdx

<Stack />
```

:::tip

> If you want a vertical stack, consider using [VStack](/docs/components/v-stack).

Reviewer: ```suggestion
If you want a vertical stack, use [VStack](/docs/components/v-stack).
```

:::

:::tip

> If you want a horizontal stack, consider using [HStack](/docs/components/h-stack).

Reviewer: ```suggestion
If you want a horizontal stack, use [HStack](/docs/components/h-stack).
```

:::

:::tip

> If you want to stack elements in depth, consider using [ZStack](/docs/components/z-stack).

Reviewer: ```suggestion
To stack child elements on top of each other, use [ZStack](/docs/components/z-stack).
```
